engine/src/CalciteInterpreter.cpp

```diff
@@ -9,7 +9,7 @@ std::shared_ptr<ral::cache::graph> generate_graph(std::vector<ral::io::data_load
 	std::vector<std::string> table_names,
 	std::vector<std::string> table_scans,
 	std::string logicalPlan,
-	int64_t connection,
+	int64_t  /*connection*/,
```

Comment:
if we are not using this parameter, we should just take it out

Reply:
Removed.